doc/Methods and Results - Rough.Rmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Methods and Results - Rough"
+author: "Farhan Majumder"
+date: "December 4, 2017"
+output: html_document
+---
+
+# Methods 
+
+## Dataset
+Data was obtained from the “Marine predator and prey body sizes”, created by Barnes et al.; joint venture generated by 17 different international institutions. As such, we have data from global locations on organisms and their environment. We focused on data on environmental variables to see which ones suggest a correlation with predator mass and prey mass. Our variables of interest were selected to be: specific habitat location and feeding behaviour. 
```

Comment:
Describe your data set in more details, at least include key columns and how the data was collected. You said it during the presentation, part of the justification of how you said up the relationship between predator and prey is how they collected the data, include this info.